translations/Marathi/research/notes.md

```diff
@@ -0,0 +1,34 @@
+# Zulip Administration
+
+# Translation - Marathi
```

Comment:
There should only be one level-one heading (a level-one heading is a heading with one #).